CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Helpers/Localization/String Resources/AnalysisStrings.swift

```diff
@@ -8,12 +8,12 @@
 import Foundation
 
 public enum AnalysisStrings: LocalizableStringResource {
-    
-    case analysisErrorMessage, documentCreationErrorMessage, cancelledMessage, loadingText, pdfPages,
+
+    case analysisErrorMessage, documentCreationErrorMessage, cancelledMessage, loadingText, loadingTextPDF, pdfPages,
     suggestion1Text, suggestion2Text, suggestion3Text, suggestion4Text, suggestion5Text, suggestionHeader,
-    defaultPdfDokumentTitle,
+    defaultPdfDokumentTitle, screenTitle,
     suggestion1Details, suggestion2Details, suggestion3Details, suggestion4Details, suggestion5Details
```

Comment:
Could you please change the way for our localization:
        title = NSLocalizedStringPreferredFormat(
            "ginicapture.help.supportedFormats.title",
            comment: "Supported formats screen title")